### DISCUSSION TITLE
Unable to finish registration for Student Pack

### DISCUSSION THREAD

**Discussion by alikebrahim** [community/community#67890] [GitHub Education] [answered]
### Hi everyone,

Hello guys,

I'm trying to register for the student pack, and when I upload my school's acceptance letter and submit for processing, the same page reloads again, but I'm not able to submit anything as the submit button gets dimmed out! I've tried to repeat the process by reloading the page and submitting again and it keeps happening to me. It doesn't seem to be sending a request as I'm not receiving any confirmation. I always end up at this situation ( after the failed first attempt to submit, the form doesn't upload the image and I have to reload the page - which only repeats the failed process) :
![application](https://github.com/community/community/assets/104144731/dae9bc87-8c15-4308-bd56-e4c2b05c2c27)

Can someone please advise

**Comment by Zerotask** [comment, marked as answer]
Did you try to use a different browser? Or your current browser with incognito mode? It could be a browser extension or some issues with a GitHub service. In the latter case, just try it another day.

If that doesn't work, please go to https://support.github.com/request/education?tags=docs-generic and contact GitHub there.

**Reply by alikebrahim** [reply]
Hey, thanks man! I tried another browser and it worked. Fingers crossed my application gets accepted

**Reply by LovitraMehta** [reply]
I'm having the same issue, I tried doing everything-different bowsers, logging out and logging in, etc. still doesn't work.

https://github.com/user-attachments/assets/b969c939-922b-4f3f-9f08-d271e3652686

**Comment by UserNhammen** [comment]
Hello there
Have this issue here:
![image](https://github.com/community/community/assets/99841123/1219f24e-b4bd-47be-a32d-29f3326942c2)

Can you help me with that?

**Reply by Zerotask** [reply]
So your GitHub user profile's full name is now the same as in your academic affiliation document? And you already logged out and in and tried to apply for it?

**Comment by dev348** [comment]
Hi, Am Trying to register for the student but my academy don't have domain the email id 
its Fully online MS degree I have the student id, enrollment letter, transcript
please help me to enroll the student pack

**Reply by Zerotask** [reply]
https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/

**Reply by dev348** [reply]
hey Hi,  Already got the approval  but still am not able to access the pack 
![image](https://github.com/community/community/assets/47909485/08881883-a313-4478-be13-016e12c226b9)
